articles/stripe-integration-react/index.md

```diff
@@ -0,0 +1,136 @@
+
+![API Test Keys](api_test_keys.jpg)
+*API Keys on Dashboard*
+### Stripe Integration using react-stripe-checkout
+
+There are many libraries available for integrating React with stripe. We will be using ```react-stripe-checkout``` library in this tutorial. 
+
+Install the library using the following command:
+
+```
+npm install react-stripe-checkout
+```
+
+Once installed, create a new component called *CheckoutWithStripe.js* and add it to your main component list. Insert the appropriate details and create your custom functions to add more details to the API call. 
+
+Here is an example function that you can modify using your custom store details:
+
+```jsx
+import React from 'react';
+import StripeCheckout from 'react-stripe-checkout';
+
+// sample function defined to compute total quantity of cart
+function computeQuantity(cart) {
+    return cart.reduce((count, itemInCart) => count + itemInCart.quantity, 0);
+}
+// similar functions can be defined to compute total price, email of the user, etc.
+
+class CheckoutWithStripe extends React.Component {
+    onToken = (res) => {
+        fetch('/save-stripe-token', {
+            method: 'POST',
+            body: JSON.stringify(token),
+        }).then(res => {
+            res.json().then(data => {
+                console.log(`Payment token generated, ${data.name}`)
+            })
+        })
+    };
+
+    render() {
+        return (
+            <StripeCheckout
+                amount = '10.00'
+                name="STRIPE_INTEGRATION"
+                // functions defined above can be used to add more information while making the API call.
+                // description={`Order of ${computeQuantity(cart)} items!`}
+                image='LINKTOIMAGE'
+                stripeKey="PUBLISHABLE_STRIPE_KEY"
+                currency="INR"
+                email='USER_EMAIL'
+                token={this.onToken}/>          
+        );
+    }
+}
+
+export default CheckoutWithStripe
+```
+
+Call the component `CheckoutWithStripe` from the payment page. 
+
+You should see a button which says `Pay with the card`.
+
+Finally, a dialog box should appear to enter card details:
+
+![Final Output](final_output.jpg)
+*Final Output*
+
+The error says that the API key is invalid. 
+
+To resolve the error, you may enter your public API key. 
+
+To verify that the application is in test mode, you will get an alert on the top right corner as shown. 
+
+### Conclusion
+
+We have walked through the entire process of accepting payments with Stripe. After setting up an account and obtaining API keys, we are able to integrate it seamlessly into our application. Stripe has made payments easy and we encourage you to build your applications with integrated payment gateways.
```

Comment:
Optionally, add an **Additional Resources** section with one or two sites. Could be the Stripe documentation or an example Stripe React project on Github for reference.